data/structure.json

```diff
@@ -0,0 +1,127 @@
+{
+  "navigation": {
```

Comment:
structure json cannot be altered, please stick to default version

Reply:
But in `structure.json` some information is lacking alts for example also I will need to add some variables to animate some components, should I hard code those, create different `.json` or modify existing `structure.json`?